### DISCUSSION TITLE
Field preset command is not setting properly

### DISCUSSION THREAD

**Discussion by LiFuji** [directus/directus#6781] [Archive]
Hello!

I filled "field presets" in creating option from "Roles and Permissions" as below:

```json
{
            "name": {
                "_eq": "$CURRENT_USER"
}
```

I used this command to preset the field "name" as the current user when creating item. So, when attempting to create an item, I left the name field blank. After saving the new item, an error came up: "The value has to be...", so the user has to set the field manually and the field has to be filled as the current user, otherwise it won't work. 

In Fields Presets, I also tried "user_created" instead, but it gave me the same response.

How can I set the name field by just creating an item?

Thanks a lot!

**Comment by benhaynes** [comment]
Not sure if that's exactly what you entered, but I believe that's malformed JSON. Missing a closing curly bracket. That's my initial observation...

**Reply by LiFuji** [reply]
Thank you for your observation, I actually didn't notice there was a curly bracket missing when pasting the command here.

**Comment by licitdev** [comment]
You've mentioned preset, you should set the value directly.

`{ "name": "$CURRENT_USER" }`

There is no need to use the `_eq` operator as that is used for filtering.

**Reply by LiFuji** [reply]
Thank you! It's working properly now!
